HDF5 XS library (#84)
arfc/moltres: python/MOLTRESXS.md

> MoltresXS

Reviewer: Strongly recommend not using capital letters for python toolkit names (or module names... or package names). PEP8 would suggest, in this case, moltresxs or, more likely: `moltres_xs`. 
```suggestion
moltres_xs
```

=====

> MoltresXS is a python tool designed for converting Serpent and SCALE 

Reviewer: ```suggestion
moltres_xs is a python tool designed for converting Serpent and SCALE 
```

> cross sections to a Moltres readable format.

Reviewer: ```suggestion
cross sections to a Moltres-readable format.
```

## Install

> MoltresXS relies on the pyne python packages.

Reviewer: There is only one PyNE.
```suggestion
moltres_xs relies on the PyNE python package.
```

> To install pyne please follow the instructions at

Reviewer: ```suggestion
To install PyNE please follow the instructions at
```

> https://pyne.io/install/index.html.

Reviewer: ```suggestion
[pyne.io](https://pyne.io/install/index.html).
```

## Use

> To use MoltresXS run the following commands

Reviewer: ```suggestion
To use moltres_xs, run the following commands.
```

```bash

> python MoltresXS.py XS.inp

Reviewer: ```suggestion
python moltres_xs.py XS.inp
```

```

## XS.inp

The cross section input file is structured such that various materials and 
branches can be condensed into a single HDF5 file. The file is broken into four sections,
[TITLE], [MAT], [BRANCH], and [FILES]. 
An example XS input file can be found in moltres/tutorial/PinCell 

#[TITLE]

> Specifies the name of HDF5 file to be created

Reviewer: Perhaps give an example for each of these. E.g.:

```ap1000_serp_xs.h5```

#[MAT]

> Species the number of materials being created.

Reviewer: ```suggestion
Specifies the number of materials being created.
```

> Lists the names of the different materials

Reviewer: Something about this MAT block documentation is incomplete.
